aerogear/mobile-security-service
# WIP: Aerogear 8375 adding ui header

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8375

## What
Adding container folder with header container file
Adding patternfly and router dependencies
Adding router to App with two routes - overview - detailed App View

## Why
See JIRA

## Verification Steps

Start the Application 
The following should be visible (so far):

-> Header with Mobile Security Service text to the left
-> Username and user icon to the right side
-> Dropdown to Logout which redirects to oauth/sign_in 
-> Which in turn redirects to Overview (for now at least)
-> Base View for App Detailed View
-> Navigation should also work if you manually enter /apps/someid

See Image
<img width="889" alt="screenshot 2019-02-14 at 17 06 36" src="https://user-images.githubusercontent.com/6498727/52803928-ef460e80-307a-11e9-85bc-fd973d591898.png">

## Checklist:
